main/states/busy_state.py

```diff
@@ -131,10 +139,15 @@ def on_exit(self):
             print("Only available for devices with RPI.GPIo ports")
 
     def __speak(self, text):
+        base_folder = os.path.dirname(os.path.abspath(__file__))
+        tts_folder = os.path.join(base_folder, '../speech/tts_wrapper.sh')
```

Comment:
I do agree with @hongquan in this respect, since it is kind of an indirect approach of invoking the tts wrapper which in turn is used to run the TTS python script, which can be achieved here only.

Moreover this indirect approach may confuse future developers as to why a shell script is being used as the middleman instead of running the python script directly.